platformio.ini

```diff
@@ -97,7 +97,7 @@ lib_ignore =
 build_flags = 
   ${common.build_flags}
   -D USER_SETUP_LOADED=1
-  -include /$PROJECT_LIBDEPS_DIR/$PIOENV/TFT_eSPI/User_Setups/Setup25_TTGO_T_Display.h
+  -include $PROJECT_LIBDEPS_DIR/$PIOENV/TFT_eSPI/User_Setups/Setup25_TTGO_T_Display.h
```

Comment:
As long as we have a fix for this PlatformIO issue, please leave this line without changes.

Reply:
ok. Done.